content/departments/product-engineering/engineering/cloud/delivery/managed/index.md

```diff
@@ -20,9 +20,19 @@ Please first read [the customer-facing managed instance documentation](https://d
 - [Enable executors][enable_executors_process.md]
 - [FAQ](#faq)
 
+## When to offer a Managed Instance
+
+Managed instances offer a backup alternative for using Sourcegraph when a customer either can't or, for some reason, won't deploy Sourcegraph self-hosted.
+
+They currently carry a heavy cost in terms of engineering time (which is not negated by the markup in cost since more money doesn't equal more time). As a result, they aren't the recommended deployment method for any tier size of customer. We hope to be able to change that in the future.
```

Comment:
I would suggest we not keep citing the cost to engineering ... I think people gloss over even though this is real and valid. What if instead of

> They currently carry a heavy cost in terms of engineering time (which is not negated by the markup in cost since more money doesn't equal more time). As a result, they aren't the recommended deployment method for any tier size of customer. We hope to be able to change that in the future.

It says

> As of 2022-03-10, managed instance is not the recommended deployment method for any tier size of customer. We hope to be able to change that in the future.

Reply:
Done 👍